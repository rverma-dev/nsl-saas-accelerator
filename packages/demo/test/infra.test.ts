import * as cdk from 'aws-cdk-lib';
import { describe } from '@jest/globals';
import { snapShotTest } from './snapshot-test';
import { DemoApprunnerStack } from '../index';

// Test prefix
const prefixDemoApprunnerStack = 'Stack(DemoApprunnerStack): ';
// const testNamePrefix3 = 'Stack(BlueprintVpc): ';

const stack = new DemoApprunnerStack(new cdk.App(), 'DemoApprunnerStack', {
  tenantID: 'string',
  deploymentId: 'string',
  deploymentType: 'string',
  deploymentTier: 'string',
  synthesizer: new cdk.DefaultStackSynthesizer({
    generateBootstrapVersionRule: false,
  }),
});
/**
 * CodeCommitRepositoryStack construct test
 */
describe('DemoApprunnerStack', () => {
  snapShotTest(prefixDemoApprunnerStack, stack);
});
