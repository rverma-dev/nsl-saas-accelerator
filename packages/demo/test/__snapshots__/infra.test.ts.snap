// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DemoApprunnerStack Stack(DemoApprunnerStack):  Snapshot Test 1`] = `
{
  "Resources": {
    "apprunner": {
      "Properties": {
        "ServiceName": "string-string-solution",
        "SourceConfiguration": {
          "ImageRepository": {
            "ImageIdentifier": "public.ecr.aws/aws-containers/hello-app-runner:latest",
            "ImageRepositoryType": "ECR_PUBLIC",
          },
        },
      },
      "Type": "AWS::AppRunner::Service",
    },
  },
}
`;
